spec/unit/defaults_spec.rb

```diff
@@ -123,4 +123,32 @@
       Puppet.settings.handlearg("--server", "test_server")
     end
   end
+
+  describe 'basemodulepath' do
+    it 'includes the user, system and vendor modules', :unless => Puppet::Util::Platform.windows? do
+      expect(
+        Puppet[:basemodulepath]
+      ).to match(%r{.*/code/modules:/opt/puppetlabs/puppet/modules:/opt/puppetlabs/puppet/vendor_modules})
+    end
+
+    describe 'on windows', :if => Puppet::Util::Platform.windows? do
+      let(:installdir) { 'C:\Program Files\Puppet Labs\Puppet' }
+
+      it 'includes packaged modules if the env_windows_installdir fact is available' do
+        Facter.stubs(:value).with(:env_windows_installdir).returns(installdir)
+
+        expect(
+          Puppet[:basemodulepath]
+        ).to match(%r{.*/code/modules;#{installdir}/puppet/modules;#{installdir}/puppet/vendor_modules})
```

Comment:
It looks like on windows we're only getting the first path `*/code/modules`, not all three as we would expect. Is it possible `Facter.value(:env_windows_installdir)` isn't returning anything for this test? If `installdir` is empty in the code, we aren't going to append `puppet/modules` or `puppet/vendor_modules` to the path
```diff
       -/.*\/code\/modules;C:\Program Files\Puppet Labs\Puppet\/puppet\/modules;C:\Program Files\Puppet Labs\Puppet\/puppet\/vendor_modules/
       +"C:/Users/appveyor/AppData/Local/Temp/1/rspecrun20180509-2900-k66xnz/tmp_settings20180509-2900-o944n5/code/modules"
```